backend/organization/views/organization_views.py

```diff
@@ -67,10 +86,62 @@
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from rest_framework.exceptions import NotFound, ValidationError
+
+from django.utils.translation import get_language
 
 logger = logging.getLogger(__name__)
 
 
+class ListOrganizationFollowersView(ListAPIView):
+    permission_classes = [IsAuthenticated]
+    serializer_class = OrganizationFollowerSerializer
+
+    def get_queryset(self):
+        return get_list_of_followers(
+            Organization, OrganizationFollower, "organization", self
+        )
+
+
+class IsUserFollowing(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request, url_slug):
+        return check_if_user_follows(
+            request.user,
+            url_slug,
+            Organization,
+            OrganizationFollower,
+            "organization",
+            "Organization not found",
+        )
+
+
+class SetFollowView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request, url_slug):
+        # probably a better way -> .mo / po files todo
+        messages = [
+            "Organization not found.",
+            "You're already following this organization.",
+            "You are now following this organization. You will be notified when they post an update!",
+            "Du folgst jetzt diese Organisation. Dir wird mitgeteilt, wenn es Updates gibt!",
+            "You weren't following this organization.",
+            "You are not following this organization anymore.",
+            "Du folgst jetzt diese Organisation nicht mehr.",
+        ]
```

Comment:
Should use mo/po files here. Also I'm not sure if it's worth it to make this generic if we need to make it so complicated with these arrays. When reading this it's really complicated to find out when which message will be printed because it's spread over multiple files. Maybe it's actually worth it to not make everything generic but just part of it

Reply:
Array is removed and the messages are not generic.